dataland-frontend/src/components/forms/parts/kpiSelection/DataPointForm.vue

```diff
@@ -168,4 +168,6 @@ export default defineComponent({
     },
   },
 });
+
+// TODO since kpiNameMappings.report is "undefined": Aren't such values framework independent and can be stored in a seperate mapping?
```

Comment:
Do the Todo

Reply:
looking into it

Reply:
Changing this now would be possible, however, it seems to me that it does not make much sense at the moment. From my point of view it has nothing to do with LkSG since data points are not used in that framework and it as part of the story when either SFDR or EU Taxonomy is changed to the new approach with data model json files. I removed the TODO.

Reply:
ok